README.rst

```diff
@@ -68,6 +68,11 @@ The quick way::
 See the install section in the documentation at
 https://docs.scrapy.org/en/latest/intro/install.html for more details.
 
+License
+=============
+
+BSD 3-clause License 
+
```

Comment:
I think it would be simpler to mention this in the first paragraph:

> Scrapy is a fast, high-level, BSD-licensed web crawling and web scraping framework

That should be enough for readers familiar with open source licenses to tell that it’s a permissive license. Anyone interested in knowing the actual flavor of BSD license can read the license and count the number of clauses in it :smile: .